Presentation.Web/app/helpers/ArrayHelper.ts

```diff
@@ -0,0 +1,30 @@
+﻿module Kitos.Helpers {
+    
+    export class ArrayHelper {
+
+        /**
+          *  Moves an element to the right hand side of another element in an array
+          * @param arr - array containing elements
+          * @param fromIndex - index of the element to be moved
+          * @param toIndex - index of the element the first element is to be moved to
+          */
+        static arrayMoveElementToRightSide(arr: any[], fromIndex: number, toIndex: number){
+            //we want to move the related column to "right" side of the base column
+            toIndex += 1;
```

Comment:
introduce a new constant in stead of modifying the input..
Also add a comment explaining why the `toIndex` is incremented by 1 since it's called `toIndex` that could cause some confusion... did you mean `rightSideOfIndex`?.. if so, rename the `toIndex` parameter to that and then introduce a constant called `toIndex` and assign it `rightSideOfIndex + 1`